Formula/mapproxy.rb

```diff
@@ -0,0 +1,37 @@
+class Mapproxy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Accelerating web map proxy"
+  homepage "https://mapproxy.org/"
+  url "https://files.pythonhosted.org/packages/98/3b/424fecff5f852100a100e326cc554df17da59d491d3c62fd305caceffcf8/MapProxy-1.14.0.tar.gz"
+  sha256 "dd36278d60cdcaaf31f7f9bbc50e90e770f3feb65cf4b3eff287215ee85f018d"
+  license "Apache-2.0"
+
+  depends_on "pillow"
+  depends_on "proj"
+  depends_on "python@3.9"
```

Comment:
Can we use `python@3.10`?

Reply:
I'd like to, but there was a problem with conflicting versions of Python, according to CI. See the first commit of this pull request for the specific error message.